src/pages/Profile.jsx

```diff
@@ -77,16 +93,8 @@ function Login() {
             className="btnBottle"
             type="submit"
             onClick={() => {
-              const url = `http://localhost:8000/users/${loginData.userId}/bottles`;
-              axios.post(url, {
-                type: typeInput.current.value,
-                appellation: appellationInput.current.value,
-                year: yearInput.current.value,
-                reference_id: 1,
-              })
-                .then((response) => {
-                  console.log(response.data);
-                });
+              handleSubmit();
+              dataSubmit();
```

Comment:
why 2 functions? even if you keep 2 API calls, you can merge the contents of handleSubmit and dataSubmit